packages/cli/test/integration.js

```diff
@@ -697,47 +765,6 @@ test('Deploy `api-env` fixture and test `vercel env` command', async t => {
     t.is(exitCode, 0, formatOutput({ stderr, stdout }));
   }
 
-  async function vcDevAndFetchCloudVars() {
```

Comment:
Are we forcing users to run `vc pull` before `vc dev` now?

Reply:
Correct. Right @leo ?

Reply:
Yes, because, even though `vercel dev` (like `vercel build`) should automatically invoke the command after prompting you (the first time), it won't invoke it every time, which makes it a potentially breaking change in my eyes, so worth a major.